dev-client/src/screens/ProjectViewScreen.tsx

```diff
@@ -15,21 +15,47 @@
  * along with this program. If not, see https://www.gnu.org/licenses/.
  */
 
+import {createContext, useContext, useCallback, useRef} from 'react';
 import ProjectTabs from 'terraso-mobile-client/components/projects/ProjectTabs';
 import {
   AppBar,
+  ScreenCloseButton,
   ScreenScaffold,
 } from 'terraso-mobile-client/screens/ScreenScaffold';
+import {BottomSheetModal, BottomSheetModalProvider} from '@gorhom/bottom-sheet';
 import {useSelector} from 'terraso-mobile-client/model/store';
+import {InfoModal} from 'terraso-mobile-client/components/common/infoModals/InfoModal';
 
 type Props = {projectId: string};
 
+const InfoPressContext = createContext({onInfoPress: () => {}});
+
+export const useInfoPress = () => useContext(InfoPressContext);
+
 export const ProjectViewScreen = ({projectId}: Props) => {
   const project = useSelector(state => state.project.projects[projectId]);
+  const infoModalRef = useRef<BottomSheetModal>(null);
+
+  const onInfoPress = useCallback(
+    () => infoModalRef.current?.present(),
+    [infoModalRef],
+  );
+  const onInfoClose = useCallback(
+    () => infoModalRef.current?.dismiss(),
+    [infoModalRef],
+  );
 
   return (
-    <ScreenScaffold AppBar={<AppBar LeftButton={null} title={project?.name} />}>
-      <ProjectTabs project={project} />
-    </ScreenScaffold>
+    <InfoPressContext.Provider value={{onInfoPress}}>
```

Comment:
question: how come this is `{{onInfoPress}}` and not `{onInfoPress}`?

Reply:
Right, I'll just pass it on its own!